cmd/keycloakb/keycloak_bridge.go

```diff
@@ -215,6 +215,13 @@ func main() {
 		}
 	}
 
+	// Security - allowed trustID groups
+	var trustIDGroups = strings.Split(c.GetString("trustid-groups"), ",")
+	var authzedTrustIDGroups = make(map[string]bool)
+	for _, grp := range trustIDGroups {
+		authzedTrustIDGroups[grp] = true
```

Comment:
I would have write this in the NewComponent(...) function... but it works too

Reply:
fixed